custom_components/tuya_local/devices/jh_pcmini.yaml

```diff
@@ -0,0 +1,26 @@
+name: Pcie computer power
+products:
+  - id: na90jncjlvw5t0ba
+    name: Mini card pcie pc power switch
+primary_entity:
+  entity: switch
+  class: outlet
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: select

```

Comment:
I wonder if this might be better as 2 buttons?

Reply:
Yes I reckon it would.
Would it be something like this?
```
secondary_entities:
  - entity: button
    name: Reset
    dps:
      - id: 101
        name: button
        type: string
        mapping:
          - dps_val: "Reset"          
  - entity: button
    name: Force reset
    dps:
      - id: 101
        name: button
        type: string
        mapping:
          - dps_val: "forceReset"
```

However with this code the reset and force reset buttons both trigger the "Reset" dps_val. I can't figure out why forceReset isn't getting triggered from the Force reset button. With the select entity dropdown it worked properly.
I've tried 
```
  - entity: button
    name: Force reset
    dps:
      - id: 101
        name: button
        type: string
        mapping:
          - dps_val: "forceReset"
            value: "forceReset"
```
but same outcome.

Reply:
`value: true` is what buttons expect

Reply:
That worked, thanks. Updated now.